components/root/root.jsx

```diff
@@ -137,6 +137,31 @@ export default class Root extends React.Component {
             e.stopPropagation();
         });
 
+        document.addEventListener('copy', (event) => {
+            const selectionArray = document.getSelection().toString().split('\n');
+
+            // Since the getSelection() doesn't take into account the CSS property 'user-select' we have to manually create a class and exclude those elements
+            const unselectableElements = document.getSelection().getRangeAt(0).commonAncestorContainer.querySelectorAll('.unselectable');
+            const unselectableElementsArray = Array.from(unselectableElements).map((element) => element.textContent);
+
+            const formattedSelection = selectionArray.filter((element) => !unselectableElementsArray.includes(element)).reduce((result, line, index) => {
```

Comment:
Non-blocking suggestion: Maybe add a comment here as to what this is filtering? I know it's looking for a post but in the future we might want to add additional use cases.

Reply:
Another thought: making copy/paste formatting pluggable would be really nice. That way, you could change the output of various post types upon copying. I may file a ticket for that 💡